pom.xml

```diff
@@ -31,7 +31,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <spotbugs.skip>true</spotbugs.skip>
     <jenkins.version>2.428</jenkins.version>
-    <selenium.version>4.11.0</selenium.version>
+    <selenium.version>4.14.0</selenium.version>
```

Comment:
what is going on here?   this is `4.14.0` yet the image it `4.14.1` - they should be identical!?
https://search.maven.org/artifact/org.seleniumhq.selenium/selenium-bom/4.14.1/pom

Reply:
> what is going on here? this is `4.14.0` yet the image is `4.14.1` - they should be identical!? https://search.maven.org/artifact/org.seleniumhq.selenium/selenium-bom/4.14.1/pom

so we are not using the bom and lift has not been released.

Reply:
only used in one place -> https://github.com/jenkinsci/acceptance-test-harness/blob/84590f923f3048068262dc43d88ed7e503c6e35e/src/main/java/org/jenkinsci/test/acceptance/po/LabelAxis.java#L22